src/api/db/models/MLModel.js

```diff
@@ -3,6 +3,20 @@ import MLModel from '../schemas/MLModel.js';
 import retry from 'async-retry';
 
 export class MLModelModel {
+  static async queryById(_id) {
```

Comment:
Do we need this? Shouldn't the method below `getMLModels({ _ids: [some_id]})` below work?